packages/native/webpack.common.js

```diff
@@ -0,0 +1,21 @@
+const { DefinePlugin } = require('webpack');
+
+const { version } = require('./package.json');
+
+const commonConfig = {
+  plugins: [
+    new DefinePlugin({
+      VERSION: JSON.stringify(version),
```

Comment:
Oh, here it is. Any reason we don't do the same thing in the `cli` package? Is there a problem importing things from `package.json` in webpack packages?

Reply:
Or perhaps a simple `Object.assign` will do?

Reply:
cli does not use webpack, it's plain node code.